ibis/backends/tests/test_generic.py

```diff
@@ -1057,6 +1057,7 @@ def test_between(backend, alltypes, df):
     backend.assert_series_equal(result, expected)
 
 
+@pytest.mark.notyet(["flink"], reason="timestamp subtraction doesn't work")
```

Comment:
Instead of xfailing, can you instead add `.drop("delta_col")` so that we still get the majority of this test on flink? It would make it so that the rest of the backends won't be tested for TimeDelta dtypes, but that should be tested more thoroughly elsewhere. More context in [cpcloud's comment here](https://github.com/ibis-project/ibis/pull/9014#issuecomment-2607166321).

Reply:
JK, I see we explicitly ADD the delta_col here in the test. xfailing sounds better.